src/include/cache_inline.h

```diff
@@ -200,14 +206,16 @@ __wt_cache_bytes_image(WT_CACHE *cache)
 static WT_INLINE uint64_t
 __wt_cache_bytes_other(WT_CACHE *cache)
 {
-    uint64_t bytes_other;
+    uint64_t bytes_other, bytes_inmem, bytes_image_intl, bytes_image_leaf;
 
+    bytes_inmem = __wt_atomic_load64(&cache->bytes_inmem);
```

Comment:
I think we may need an interface for the cache which returns atomic loads and stores, or it locks every access effectively getters an setters. I think we can do it with function pointers, I'll think on it.

Reply:
Technically private interfaces are possible but they're not coming to WiredTiger without mega refactors so I'll close this comment.